cms/static/build-debug.js

```diff
@@ -0,0 +1,158 @@
+                deps: [
+                    'jquery', 'underscore', 'codemirror', 'tinymce',
+                    'jquery.tinymce', 'jquery.qtip', 'jquery.scrollTo', 'jquery.flot',
+                    'jquery.cookie', 'utility'
+                ]
+            }
+        },
+        /**
+         * Introduced in 2.1.2: If using "dir" for an output directory, normally the
+         * optimize setting is used to optimize the build bundles (the "modules"
+         * section of the config) and any other JS file in the directory. However, if
+         * the non-build bundle JS files will not be loaded after a build, you can
+         * skip the optimization of those files, to speed up builds. Set this value
+         * to true if you want to skip optimizing those other non-build bundle JS
+         * files.
+         */
+        skipDirOptimize: true,
+        /**
+         * When the optimizer copies files from the source location to the
+         * destination directory, it will skip directories and files that start
+         * with a ".". If you want to copy .directories or certain .files, for
+         * instance if you keep some packages in a .packages directory, or copy
+         * over .htaccess files, you can set this to null. If you want to change
+         * the exclusion rules, change it to a different regexp. If the regexp
+         * matches, it means the directory will be excluded. This used to be
+         * called dirExclusionRegExp before the 1.0.2 release.
+         * As of 1.0.3, this value can also be a string that is converted to a
+         * RegExp via new RegExp().
+         */
+        fileExclusionRegExp: /^\.|spec|spec_helpers/,
+        /**
+         * Allow CSS optimizations. Allowed values:
+         * - "standard": @import inlining and removal of comments, unnecessary
+         * whitespace and line returns.
+         * Removing line returns may have problems in IE, depending on the type
+         * of CSS.
+         * - "standard.keepLines": like "standard" but keeps line returns.
+         * - "none": skip CSS optimizations.
+         * - "standard.keepComments": keeps the file comments, but removes line
+         * returns.  (r.js 1.0.8+)
+         * - "standard.keepComments.keepLines": keeps the file comments and line
+         * returns. (r.js 1.0.8+)
+         * - "standard.keepWhitespace": like "standard" but keeps unnecessary whitespace.
+         */
+        optimizeCss: 'none',
+        /**
+         * How to optimize all the JS files in the build output directory.
+         * Right now only the following values are supported:
+         * - "uglify": Uses UglifyJS to minify the code.
+         * - "uglify2": Uses UglifyJS2.
+         * - "closure": Uses Google's Closure Compiler in simple optimization
+         * mode to minify the code. Only available if REQUIRE_ENVIRONMENT is "rhino" (the default).
+         * - "none": No minification will be done.
+         */
+        optimize: 'none',
```

Comment:
Yeah, that would be much better. As much as I hate to say it, could you use an environment variable, or something?